### PR TITLE
removing tutorial form the repository (closes #238)

### LINKED ISSUE #238: review jupyter packages dependencies
it used to be issue with asyncio and I had to downgrade the packages (tornado)

### PR DESCRIPTION
**Pull request by djarecka**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Summary

tutorial will be in a separate repository

## Checklist

- [ ] All tests passing
- [ ] I have added tests to cover my changes
- [ ] I have updated documentation (if necessary)
- [ ] My code follows the code style of this project
(we are using `black`: you can `pip install pre-commit`,
run `pre-commit install` in the `pydra` directory
and `black` will be run automatically with each commit)

## Acknowledgment
- [ ] I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

**Comment by satra** [comment]
please make sure to update the readme to point to the correct tutorial location.

**Comment by djarecka** [comment]
yes, just want to do the new repo first, should mark as wip

**Comment by djarecka** [comment]
@satra - updated README and also decided to create `tutorial/README` in case someone tries to use the old link

File: tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Pydra Tutorial
+
+Python Tutorial has been moved to a separate [GitHub repository](https://github.com/nipype/pydra-tutorial).
+
+The tutorial could be run with [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/nipype/pydra-tutorial/master?filepath=notebooks)
```
Comment (satra) [inline tutorial/README.md:5]:
```suggestion
The interactive tutorial is available at  [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/nipype/pydra-tutorial/master?filepath=notebooks)
```

**Pull request merged**